```
> Task :react-native-ledger-core:compileReleaseJavaWithJavac FAILED
/Users/grenaudeau/dev/ledger-live-mobile/node_modules/@ledgerhq/react-native-ledger-core/android/src/main/java/com/ledger/reactnative/RCTCoreExecutionContext.java:55: error: constructor ExecutionContextImpl in class ExecutionContextImpl cannot be applied to given types;
        ExecutionContextImpl newInstance = new ExecutionContextImpl(this.reactContext);
                                           ^
  required: ReactApplicationContext,int
  found: ReactApplicationContext
  reason: actual and formal argument lists differ in length
1 error
```

--- a/android/src/main/java/com/ledger/reactnative/ExecutionContextImpl.java
+++ b/android/src/main/java/com/ledger/reactnative/ExecutionContextImpl.java
@@ -21,6 +21,10 @@ public class ExecutionContextImpl extends co.ledger.core.ExecutionContext {
         pool = Executors.newFixedThreadPool(numberOfThreads);
     }
 
+    public ExecutionContextImpl(ReactApplicationContext reactContext) {
+        this(reactContext, 1);
+    }
+
 //    @Override
 //    protected void finalize() throws Throwable {
 //        super.finalize();
@@ -36,7 +40,6 @@ public class ExecutionContextImpl extends co.ledger.core.ExecutionContext {
      *Execute a given runnable
      *@param runnalbe, Runnable object
      */
-
     public void execute(final co.ledger.core.Runnable runnable) {
         pool.execute(new java.lang.Runnable() {
             @Override

